### PR TITLE
[AUD-21] Add delegation Dashboard Chip

### PR DESCRIPTION
**Pull request by piazzatron**

### CODE REVIEW

File: src/store/cache/user/hooks.ts [hunk 1]
```diff
@@ -355,6 +355,23 @@ export const useTotalDelegates = ({ wallet }: UseTotalDelegatesProps) => {
   return { status, totalDelegates }
 }
 
+/** Returns the total amount delegated inbound to an SP */
+export const useTotalInboundDelegation = ({ wallet }: { wallet: Address }) => {
```
Comment (raymondjacobson) [inline src/store/cache/user/hooks.ts:359]: useActiveInboundDelegation (total is a scary words that has lost meaning i think)

**Review by raymondjacobson — APPROVED** [review]

File: src/store/cache/user/hooks.ts (hunk 1 above)
Comment (piazzatron) [reply src/store/cache/user/hooks.ts:359]: will change

Comment (raymondjacobson) [reply src/store/cache/user/hooks.ts:359]: or maybe useSummedActiveInboundDelegation? i get that you'd wanna capture the "totaling" of it

Comment (piazzatron) [reply src/store/cache/user/hooks.ts]: I think summed is implied by your OG suggestion tbh

**Pull request merged**